tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -187,6 +187,65 @@ Save data block
 
+    user clicks button    Save data block
+    user waits until page contains    Delete this data block
+
+Validate data block is in list
+    user clicks link    Back
+    user waits until h2 is visible    Data blocks
+
+    user waits until table is visible
+    user checks table column heading contains    1    1    Name    testid:dataBlocks
+    user checks table column heading contains    1    2    Has chart    testid:dataBlocks
+    user checks table column heading contains    1    3    In content    testid:dataBlocks
+    user checks table column heading contains    1    4    Created date    testid:dataBlocks
+    user checks table column heading contains    1    5    Actions    testid:dataBlocks
+
+    user checks table body has x rows    1    testid:dataBlocks
+    user checks table cell contains    1    1    ${DATABLOCK_NAME}    testid:dataBlocks
+    user checks table cell contains    1    3    No    testid:dataBlocks
+
+Start creating a featured table
+    user clicks link    Create data block
+    user waits until table tool wizard step is available    1    Select a data set
+
+Select subject "UI test subject"
+    user waits until page contains    UI test subject    %{WAIT_SMALL}
+    user clicks radio    UI test subject
+    user clicks element    id:publicationDataStepForm-submit
+    user waits until table tool wizard step is available    2    Choose locations    %{WAIT_MEDIUM}
+    user checks previous table tool step contains    1    Data set    UI test subject
+
+Select locations
+    user opens details dropdown    Opportunity area
+    user clicks checkbox    Bolton 001
+
+    user clicks element    id:locationFiltersForm-submit
+    user waits until table tool wizard step is available    3    Choose time period    90
+
+Select time period
+    user waits until page contains element    id:timePeriodForm-start
+    user chooses select option    id:timePeriodForm-start    2009
+    user chooses select option    id:timePeriodForm-end    2017
+    user clicks element    id:timePeriodForm-submit
+    user waits until table tool wizard step is available    4    Choose your filters
+    user checks previous table tool step contains    3    Time period    2009 to 2017    %{WAIT_MEDIUM}
+
+Select indicators
+    user checks indicator checkbox is checked    Admission Numbers
+
+Create table
+    [Documentation]    EES-615
+    user clicks element    id:filtersForm-submit
+    user waits until results table appears    %{WAIT_LONG}
+    user waits until element contains    testid:dataTableCaption
+    ...    Admission Numbers for 'UI test subject' in Bolton 001 between 2009 and 2017
+
+Save data block
+    user enters text into element    id:dataBlockDetailsForm-name    UI test featured table
+    user enters text into element    id:dataBlockDetailsForm-heading    UI test featured table title
+    user enters text into element    id:dataBlockDetailsForm-source    UI test featured table source
```

Comment:
Could use the more human-readable `label` selectors e.g. `label:Name`. Would want to do it in a few other places in this test suite as well.